apps/entity-service/src/entities/dto/entity-query.dto.ts

```diff
@@ -4,7 +4,7 @@ import { NumberPage } from "@terramatch-microservices/common/dto/page.dto";
 
 class QuerySort {
   @ApiProperty({ name: "sort[field]", required: false })
-  field: string;
+  field?: string;
```

Comment:
It's obnoxious, but the query param validation system and the documentation generation system don't always sync up as I'd like. You've made the param not required from a documentation perspective, but I believe you need to add `@IsOptional()` to avoid problems with validation.

Reply:
added